### PR TITLE
feat: sepolia default

### PR DESCRIPTION
**Pull request by alaibe**

### CODE REVIEW

File: appdatabase/migrations/sql/1708416025_make_sepolia_default.up.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE settings RENAME COLUMN is_sepolia_enabled TO is_goerli_enabled;
```
Comment (saledjenic) [inline appdatabase/migrations/sql/1708416025_make_sepolia_default.up.sql]: @alaibe does this work on iOS? I remember in the past we couldn't do it that way.

**Review by saledjenic — APPROVED** [review]

File: appdatabase/migrations/sql/1708416025_make_sepolia_default.up.sql (hunk 1 above)
Comment (rasom) [reply appdatabase/migrations/sql/1708416025_make_sepolia_default.up.sql]: yep as I remember the only way to rename columns is to copy table to another temporary table, create a new one with necessary fields, copy records from temp table and then remove temp table

Comment (smohamedjavid) [reply appdatabase/migrations/sql/1708416025_make_sepolia_default.up.sql]: Yes, iOS gets the following error (on login) `failed to apply status-go/protocol migrations: failed to migrate: file does not exist` @alaibe 

 EDIT: We need to do the way Roman as suggested

Comment (alaibe) [reply appdatabase/migrations/sql/1708416025_make_sepolia_default.up.sql]: Alright, i didn't know iOS was special, fixing that

**Pull request merged**